### DISCUSSION TITLE
docker pull with rancher desktop: unable to fetch right registry how can I fix this ?

### DISCUSSION THREAD

**Discussion by skcloud2007** [rancher-sandbox/rancher-desktop#6562] [Q&A]
Error response from daemon: unauthorized: image **registry-1.docker.io**/kiwigrid/k8s-sidecar:XX.XX is not covered by the Rancher Desktop allowed-images list.

While it should fetch **docker.io**/kiwigrid/k8s-sidecar
Please suggest some solution.

**Comment by mook-as** [comment]
If you're trying to pull [`kiwigrid/k8s-sidecar`](https://hub.docker.com/r/kiwigrid/k8s-sidecar), the default registry is Docker Hub. That has the internal address `registry-1.docker.io` (because they started out as `docker.io`, and then later got the `docker.com` domain).  For the purposes of the allowed images list this is fine because we internally map patterns with no registry to Docker Hub for compatibility.

If you prefer using a different registry, you'll need to specify that.

Note that if you're filtering by tag, please be aware that [currently there are issues](https://docs.rancherdesktop.io/ui/preferences/container-engine/allowed-images#how-to-specify-allowed-image-patterns); the tag will be allowed, but the digest behind the tag will be rejected.
